inbm/dispatcher-agent/dispatcher/aota/application_command.py

```diff
@@ -212,13 +246,16 @@ def __init__(self,  dispatcher_broker: DispatcherBroker,
         super().__init__(dispatcher_broker, parsed_manifest, dbs, update_logger=update_logger)
 
     def update(self):  # pragma: no cover
-        super().update()
+        super().update()        
         application_repo = self._download_package()
-        install_cmd = application_repo.get_repo_path() + "/" + self.resource if self.resource else ""
-        if ' ' in install_cmd or install_cmd.isspace():
-            logger.debug(f"INSTALL : {install_cmd}")
-            raise AotaError(f"File path cannot contain spaces - {install_cmd}")
-        base_command = f"/usr/bin/dpkg -i {install_cmd}"
+        path_to_pkg = application_repo.get_repo_path() + "/" + self.resource if self.resource else ""
```

Comment:
**Suggestion:** Use os.path.join for constructing file paths to ensure compatibility across different operating systems. [best practice]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        path_to_pkg = os.path.join(application_repo.get_repo_path(), self.resource) if self.resource else ""
```